test/python_bindings.py

```diff
@@ -67,6 +67,9 @@ def testConstants(self):
     def testIdealGas(self):
         eos = singularity_eos.IdealGas(1,1)
 
+    def testStiffGas(self):
+        eos = singularity_eos.StiffGas(1,1,1)
```

Comment:
Can you add the last argument?
```suggestion
        eos = singularity_eos.StiffGas(1,1,1,1)
```